R/sem.R

```diff
@@ -1071,11 +1117,11 @@ checkLavaanModel <- function(model, availableVars) {
     gettext("Expected cross validation index (ECVI)")
   )
   if (length(options[["models"]]) == 1) {
-    fitot[["value"]] <- fmli[[1]][c("rmsea", "rmsea.ci.lower", "rmsea.ci.upper", "rmsea.pvalue",
+    fitot[["value"]] <- fmli[[1]][c("rmsea", "rmsea.ci.lower", "rmsea.ci.upper", "rmsea.pvalue", "rmsea.t",
```

Comment:
can you add citations of the original T-size articles to the tables that report these measures?